docs-src/content/using-cubic-sdk/client-configs.md

```diff
@@ -0,0 +1,108 @@
+
+This page here discusses the more common combinations sent to the server.
+
+## Fields
+
+Here's a quick refresher of the fields.
+
+| Field | Required | Default | Description |
+| ----- | -------- | ------- | ----------- |
+| model_id | Yes | | Unique ID of the model to use. |
+| audio_encoding | Yes | | Encoding format of the audio, such as RAW_LINEAR_16, WAV, MP3, etc. |
+| idle_timeout | No | `0s` (Unlimited) | Maximum time allowed between each gRPC message. The server may place further restrictions depending on its configuration. |
+| enable_word_time_offsets | No | `false` | Toggles the calculation of word-level timestamps.  Cubic models must support word level timestamps for this field to be populated. |
+| enable_word_confidence | No | `false` | Toggles the calculation of word-level confidence scores. Cubic models must support word level confidence for this field to be populated.|
+| enable_raw_transcript | No | `false` | If `true`, the raw transcript will be included in the results. |
+| enable_confusion_network | No | `false` | Toggles the creation of a confusion network, consisting of multiple alternatives transcriptions.  Cubic models must support confusion networks for this field to be populated. |
+| audio_channels | No | `[0]` (mono) | Specifies which channels of a multi-channel audio file to be transcribed, each as their own individual audio stream. |
+
+## Use cases
+
+The most basic use case is getting a formatted transcript.  This would simply need a config such as:
+
+```json
+{
+    "model_id": "1",
+    "audio_encoding": "raw"
+}
+```
+
+and your resulting transcript would be found at `results.alternatives[*].transcript`.
+
+> Note: Prior to `cubicsvr` v2.9.0 and SDK-Cubic v1.3.0, the field `results.alternatives[*].transcript` was populated with either the raw or the formatted transcription depending on the `enable_raw_transcript` config.  After these changes, raw transcripts have been pushed to a new field `results.alternatives[*].raw_transcript`, and only populated when `enable_raw_transcript` is set to true.
+
+------
+
+Sometimes, only the Raw transcript is desired.  In this case, there are two options.  
+
+1. Disable server-side formatting and use the above config.  Retrieve the raw transcript from the `results.alternatives[*].transcript` field.
+2. Specify the `enable_raw_transcript = true` flag, and access the field `results.alternatives[*].raw_transcript`.
+
+> Note: Prior to `cubicsvr` v2.9.0 and SDK-Cubic v1.3.0, the field `results.alternatives[*].transcript` was populated with either the raw or the formatted transcription depending on the `enable_raw_transcript` config.  After these changes, raw transcripts have been pushed to a new field `results.alternatives[*].raw_transcript`, and only populated when `enable_raw_transcript` is set to true.
```

Comment:
Do we need this notice duplicated so close to the first notice?

Reply:
I don't think so.  I think we should have it only here and remove the previous one.